src/feed-worker.js

```diff
@@ -7,8 +7,10 @@ exports.start = function () {
   // Start processing jobs from the feed queue...
   feedQueue.process(async (job) => {
     const { url } = job.data;
+
     console.log(`Processing job - ${url}`);
     // For now, just get the feed data and dump to the console
     return request(url).then((data) => console.log(`${data} + '\n\n'`)).catch((err) => { throw err; });
   });
```

Comment:
It looks like the last version has been overridden once because the original version was this:
```javascript
return request(url).then((data) => console.log(`${data}\n\n`));
```
and actually there is nothing wrong with it. So I suggest to not change this function. If you want to catch from async function you need to do in try and catch block rather than a promise like catch.